media-api/app/controllers/MediaApi.scala

```diff
@@ -413,7 +413,9 @@ class MediaApi(
   def imageSearch() = auth.async { request =>
     implicit val r = request
 
-    val shouldFlagGraphicImages = request.cookies.get("SHOULD_BLUR_GRAPHIC_IMAGES").exists(_.value == "true")
+    // defaults to true if not specified
```

Comment:
Does it not default to `defaultShouldBlurGraphicImages` if not specified?

Reply:
got spot, thanks. this was an old comment from before it was made configurable - I've removed the comment now as the code is self-explanatory with `.getOrElse(config.defaultShouldBlurGraphicImages`